core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -894,6 +896,10 @@ object AutoTuner extends Logging {
   val MAX_PINNED_MEMORY_MB: Long = 4 * 1024L
   // Default pinned memory to use per executor in MB
   val DEF_PINNED_MEMORY_MB: Long = 2 * 1024L
+  // Default pageable pool to use per executor in MB
+  val DEF_PAGEABLE_POOL_MB: Long = 1 * 1024L
+  // Default spill storage to use per executor in MB
+  val DEF_SPILL_STORAGE_MB: Long = 1 * 1024L
```

Comment:
spill storage isn't 1G by default, its combined size of pinned and pageable . This getscomplicated though because it can use space from pinned and pageable.

Reply:
Removed spill storage in calculations in latest commit